pkg/systemd/terminal.jsx

```diff
@@ -51,6 +57,13 @@ const _ = cockpit.gettext;
             this.setState({ title: title });
         }
 
+        onThemeChanged(value) {
+            this.setState({ theme: value });
+            var cookie = "CockpitTerminalTheme=" + encodeURIComponent(value) +
+                         "; path=/; expires=Sun, 16 Jul 3567 06:23:41 GMT";
+            document.cookie = cookie;
```

Comment:
Shouldn't that be `theme_cookie` or so, to be more specific? Note that we already have a login session cookie.

Reply:
Language selection is saved in cookie `CockpitLang`

Reply:
Right, but when introspecting `document` or just for avoiding silly name conflicts, it'd still be better to give it a more specific name.

Reply:
Nevermind me, `document.cookie` is a standard API.